content/issues/2021-11-25-maintenance-k8s.md

```diff
@@ -0,0 +1,21 @@
+---
+title: AKS publick8s version update
+date: 2021-11-25T08:00:00-00:00
+resolved: false
+# resolvedWhen: 2021-06-17T09:45:00-00:00
+# Possible severity levels: down, disrupted, notice
+severity: notice
+affected:
+  - publick8s
+  - cik8s
```

Comment:
```suggestion
affected:
  - accounts.jenkins.io
  - beta.accounts.jenkins.io
  - customize.jenkins.io
  - get.jenkins.io
  - incrementals.jenkins.io
  - infra.ci.jenkins.io
  - javadoc.jenkins.io
  - jenkins-wiki-exporter.jenkins.io
  - ldap.jenkins.io
  - plugins.jenkins.io
  - release.ci.jenkins.io
  - reports.jenkins.io
  - www.jenkins.io
  - uplink.jenkins.io
```
As per the previous upgrade https://raw.githubusercontent.com/jenkins-infra/status/main/content/issues/2021-10-25-maintenance-window-aks.md.
The rationale is that `cik8s` and `publick8s` are names used internally to the infra team, while the end user expects to know the service that could be impacted (this was a feedback from @daniel-beck a few month ago if I understood it correctly).